diff --git a/recipes-support/python3-jmespath/python3-jmespath_0.10.0.bb b/recipes-support/python3-jmespath/python3-jmespath_0.10.0.bb
index f290fe4..1430e17 100644
--- a/recipes-support/python3-jmespath/python3-jmespath_0.10.0.bb
+++ b/recipes-support/python3-jmespath/python3-jmespath_0.10.0.bb
@@ -2,11 +2,11 @@ SUMMARY = "JMESPath"
 DESCRIPTION = ""
 HOMEPAGE = ""
 LICENSE = "MIT"
-LIC_FILES_CHKSUM = "file://LICENSE;md5=2683790f5fabb41a3f75b70558799eb4"
+LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=2683790f5fabb41a3f75b70558799eb4"
 
 inherit setuptools3
 
-SRC_URI = "git://github.com/jmespath/jmespath.py.git;tag=v1.0.2"
+SRC_URI = "git://github.com/jmespath/jmespath.py.git;tag=0.10.0"
 
 S = "${WORKDIR}/git"
 
